modules/distribution/src/bin/adaptive.bat

```diff
@@ -0,0 +1,100 @@
+@echo off
+rem ----------------------------------------------------------------------------
+rem Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com).
+rem
+rem WSO2 Inc. licenses this file to you under the Apache License,
+rem Version 2.0 (the "License"); you may not use this file except
+rem in compliance with the License.
+rem You may obtain a copy of the License at
+rem
+rem http://www.apache.org/licenses/LICENSE-2.0
+rem
+rem Unless required by applicable law or agreed to in writing,
+rem software distributed under the License is distributed on an
+rem "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+rem KIND, either express or implied.  See the License for the
+rem specific language governing permissions and limitations
+rem under the License.
+
+rem ----------------------------------------------------------------------------
+rem Adaptive Authentication Script for the WSO2 Carbon Server
+
+rem Environment Variable Prequisites
+
+rem    CARBON_HOME   	Home of WSO2 Carbon installation. If not set I will  try
+rem                    	to figure it out.
+rem    NASHORN_VERSION   	OpenJDK Nashorn Version

```

Comment:
indentation mismatch. Lets start the variable description at the same indentation level as other variable descriptions